buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -14,4 +14,6 @@ object Versions {
     const val JSON_WEBTOKEN = "0.11.5"
     const val JJWT_IMPL = "0.11.5"
     const val JJWT_JACKSON = "0.11.5"
+    const val CIRCUIT_BREAKER = "3.0.4"
+    const val ACTUATOR = "3.2.3"
```

Comment:
Why do you need actuator ?

Reply:
to see the status of server and other settings in real time

Reply:
Do you need it for circuit breaker ? Or just for yourself ?

Reply:
just for me